site/src/atomics/width.md

```diff
@@ -118,4 +130,6 @@ List of all available classes:
 .width-450-desktop
 .width-500-desktop
 .width-unset-desktop
+.max-width-full
+.max-width-100vw
```

Comment:
[nit] should we move these to the top of the list? Alphabetically and so it doesn't get lost after the responsive class names?